framework-profile.yaml

```diff
@@ -15,6 +15,15 @@ flavors:
   ubuntu:
     - systemd-base
     - dracut-network-legacy
+  ubuntu-arm-rpi:
+    - systemd-base
```

Comment:
we should probably need to start grouping this below and just have a single "metapackage" here to not repeat ourselves this much.

Reply:
do you mean using anchors? or what is a meta package?

Reply:
no, see how you mark here `systemd-base` and then below you can see that it contains 2 packages: `init-svc/systemd` and `systemd-default/network` ? So Im thinking that we may need to just merge more of the stuff seeing that there is a clear repetition here with systemd-base and dracut-network-legacy*

so something similar to those, like ubuntu-base that includes all those packages, or something shared that includes all of that so its a one line for all. Dont know how, just thinking out loud seeing that this keeps frowing and lines repeating :D

Reply:
Ok, I get your point now. Yeah I agree, I had the same feeling that this was lots of duplication but didn't know how this part integrates to be able to suggest a recommendation. I'll create an issue out of your comment.